src/librustdoc/html/render.rs

```diff
@@ -992,13 +992,11 @@ themePicker.onblur = handleThemeButtonsBlur;
             writeln!(v, "{}", *implementor).unwrap();
         }
         v.push_str(
-            r"
-            if (window.register_implementors) {
-                window.register_implementors(implementors);
-            } else {
-                window.pending_implementors = implementors;
-            }
-        ",
+            "if (window.register_implementors) {\
+                 window.register_implementors(implementors);\
+             } else {\
+                 window.pending_implementors = implementors;\
+             }",
```

Comment:
This change makes it harder to read the string.  Why did it happen?

Reply:
To be more consistent with the rest of the code. Everywhere else, we inline the output HTML/JS to reduce its size.

Reply:
As with the other, it's that the change from a raw string to a non raw string means those backslashes are a pain.